device-types/HPE/HPE-SY480.yaml

```diff
@@ -0,0 +1,6 @@
+manufacturer: HPE
+model: HPE-SY480
+slug: HPE-SY480
+u_height: 0
```

Comment:
I am confused by this model.  From the docs it looks like almost the form factor of a blade (from a blade chassis). I guess you would line them up on a rack shelf?

Reply:
This is the blade yes, you put them in the parent chassis.